advanced_source/cpp_cuda_graphs/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
```

I don't think it will actually compiles with 3.1
```suggestion
cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```